packages/@okta/vuepress-site/docs/guides/pwd-optional-change-email/main/react/allcontent.md

```diff
@@ -0,0 +1,52 @@
+Enable users to change their primary email address in an application based on the embedded SDK.
+
+**Learning outcomes**
+
+* Add the ability for a user to change their primary email in your SPA app using the Embedded SDK and MyAccount API.
+
+**What you need**
+
+<StackSnippet snippet="whatyouneed" />
+</br>
+
+**Sample code**
+
+<StackSnippet snippet="samplecode" />
+
+---
+
+## Overview
+
+Allowing a user to change their primary email is a critical task when they only use their username and email address to sign in to your app. Much like a password, a user must be able to change their email address, especially if it becomes compromised. With the [Embedded SDK](https://github.com/okta/okta-auth-js) you can enable your users to change their primary emails in your app.
```

Comment:
I think Embedded SDK makes sense when it's common content shared between all SDKs, since this content is not shared I changed it to Okta Auth Javascript SDK.